### ISSUE TITLE
Mismatch bcd key for `url()` function

### ISSUE THREAD

**Issue by skyclouds2001** [mdn/browser-compat-data#25413]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/url_function

### What specific section or headline is this issue about?

Browser compatibility

### What information was incorrect, unhelpful, or incomplete?

the bcd key refers to the `<url>` css type, not the `url()` function

### What did you expect to see?

fix the bcd key

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/url_function`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/url_function
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/url_function/index.md
* Last commit: https://github.com/mdn/content/commit/919d97a4bda8004f63f655d3f9576c27a82c8a2a
* Document last modified: 2024-11-19T12:34:07.000Z

</details>

**Comment by estelle** [comment]
> @estelle Is the documentation of the [`<url>`](https://developer.mozilla.org/en-US/docs/Web/CSS/url_value) type in any way useful - I mean does this actually every appear in "code" or do you only ever see `url()`? If not, it might make sense to not document it and point to the URL function everywhere.

What @skyclouds2001 said.
That page as written is useless, but the page is needed. we could do a much better job 
For example "The value is either of the following:" and then there is only one value?!?!
We do document (or try to document) all the data types, and `<url>` is a data type, so must be documented. It is found in multiple property values, so needs its own page (there are some data types that are only on the property page for which they are a value)

**Comment by hamishwillee** [comment]
Great. So I guess the first thing is (still) to get a feature for the `url()` function and look at the docs for this as a separate problem.

**Comment by skyclouds2001** [comment]
> For example "The value is either of the following:" and then there is only one value?!?!

I guess this is because previously there is also a `src()` function, but since it is not supported by any browsers at present, it was removed via https://github.com/mdn/content/pull/36839.

**Comment by skyclouds2001** [comment]
> Great. So I guess the first thing is (still) to get a feature for the function and look at the docs for this as a separate problem.`url()`

yes, and I guess the css `url()` function should have the same bcd info as the css `<url>` type

**Comment by caugner** [comment]
For consistency, we should probably:

1. Change the description of `css.types.url` to `url()` (same for `css.types.symbols`: `symbols()`).
2. Set the `mdn_url` to [Web/CSS/url_function](https://developer.mozilla.org/en-US/docs/Web/CSS/url_function) (not [Web/CSS/url_value](https://developer.mozilla.org/en-US/docs/Web/CSS/url_value)).

Currently, the https://developer.mozilla.org/en-US/docs/Web/CSS/url_value page has no BCD table, and it would probably only make sense to add one once browsers support `src()` as well: Then we could show both `css.types.src` and `css.types.url`.
